capstone/src/main/webapp/dashboard.html

```diff
@@ -67,7 +67,7 @@ <h5 class="modal-title" id="exampleModalCenterTitle">Pick A Room Title</h5>
     <div class="container-fluid">
         <div id="room-container" class="row row-cols-1 row-cols-sm-4"></div>
     </div>
- 
+
```

Comment:
Cloud shell was being weird about this white space and I couldn't figure out how not to commit it sorry.